cdap-tms/pom.xml

```diff
@@ -161,7 +161,7 @@
     <dependency>
       <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-storage-spi</artifactId>
-      <version>6.1.2-SNAPSHOT</version>
```

Comment:
not from this PR, but this should use `${project.version}`

Reply:
Done.